docker-compose.yml

```diff
@@ -33,6 +33,7 @@ services:
       - 5555:5432
   api:
     container_name: ump-api
+    env_file: .env
```

Comment:
Do we really need this here? :thinking:

Reply:
There are a lot of configs for the api in the .env file. So i think yes.